extensions/index.json

```diff
@@ -8,7 +8,7 @@
 		"mailchimp",
 		"map",
 		"markdown",
-		"membership-button",
+		"recurring-payments",
```

Comment:
You'd need to reorder the list to keep it alphabetically organized I think.

Reply:
Changed!